pulseapi/profiles/test_views.py

```diff
@@ -25,6 +26,11 @@
 from pulseapi.profiles.factory import UserBookmarksFactory, ExtendedUserProfileFactory
 
 
+# Converts the JSON String to a named tuple - The name of the tuple (X) is unimportant
+def json_to_obj(data):
+    return json.loads(data, object_hook=lambda data: namedtuple('X', data.keys())(*data.values()))
```

Comment:
not sure I understand this. Why do we want a named tuple?

Reply:
honestly, not sure we do? I looked for a method of converting JSON to a python object, this is what I found. let me try with tuple.